README.md

```diff
@@ -1,24 +1,188 @@
+    </ul>
+</details>
+<details>
+  <summary>Frontend</summary>
+    <ul>
+      <li><a href="https://reactjs.org/">React</a></li>
+    </ul>
+</details>
+<details>
+  <summary>Database</summary>
+    <ul>
+      <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Establishing a connection between a Ruby on Rails backend and a React frontend** that involves creating a bridge that allows these two technologies to communicate and work together seamlessly.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Make sure you have Ruby and Ruby on Rails installed on your computer before running the project. To check if you have Ruby installed, open your console and run the command ruby -v.
+
+If Ruby is not installed, follow the instructions below based on your operating system:
+
+Windows
+Follow the [tutorial](https://gorails.com/setup/windows/10) for installing Ruby on Windows.
+
+MacOS
+Run the following commands to install Ruby using Homebrew:
+
+```sh
+  brew install rbenv ruby-build
+  echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+  source ~/.bash_profile
+
+  # Install Ruby
+  rbenv install 3.0.1
+  rbenv global 3.0.1
+  ruby -v
+```
+
+To install Ruby on Rails, run:
+
+```sh
+  gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/federicaulzurrun/hello-rails-back-end.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install 
+```
+
+### Usage
+
+```sh
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+### Deployment
+To deploy the project locally, run the following command:
+
+```sh
+  rails s
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Federica Ulzurrun**
+
+- GitHub: [@federicaulzurrun](https://github.com/federicaulzurrun)
+- Twitter: [@ulzurrunfede](https://mobile.twitter.com/ulzurrunfede)
+- LinkedIn: [Federica Ulzurrun](https://www.linkedin.com/in/federicaulzurrun/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Developing the UI]**
+- [ ] **[Deploying the site]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/federicaulzurrun/SchoolLibrary/issues).
+
```

Comment:
- [ ] It would be nice if you put the right link here because this link is taking us to the School Library issues page 😄 Kindly put the link of this repo issues page to make your readme descriptive and professional 👍